clients/admin-ui/src/features/data-discovery-and-detection/tables/cells/DetectionItemActionsCell.tsx

```diff
@@ -78,21 +81,43 @@ const DetectionItemActionsCell = ({
           loading={confirmIsLoading}
         />
       )}
-      {showUnmuteAction && (
+      {showUnMuteAction && (
         <ActionButton
-          title="Monitor"
+          title="Un-Mute"
```

Comment:
Small nitpick: I don't think [unmute](https://en.wiktionary.org/wiki/unmute) should be hyphenated.